syntax = "proto3";

package rules;

message AssertTrendRequest {
    string symbol = 1;
    int32 lfast = 2;
    int32 lslow = 3;
    bool use_attenuation = 4;
    float scaling_factor = 5;
    string scaling_type = 6;

}

message AssertTrendResponse {
    bytes series = 1;
}

service AssertTrend{
    rpc get_asserttrend (AssertTrendRequest) returns (AssertTrendResponse);
}